src/cmd.rs

```diff
@@ -355,6 +356,7 @@ impl CommandExtension for Finalize {
             network,
         );
 
+        println!("Anchoring address is {}", address);
```

Comment:
Why not using `log`?

Reply:
This information is very useful for end user despite of logging verbosity.